app/src/main/res/values/dimens.xml

```diff
@@ -39,16 +44,29 @@
     <dimen name="normal_lse">6sp</dimen>
     <dimen name="big_lse">12sp</dimen>
 
+    <!--    view radius-->
+    <dimen name="search_view_corner">20dp</dimen>
+<!--    todo-->
```

Comment:
why you write todo here?
maybe you forgot to change something

Reply:
fixed